tiatoolbox/dataloader/slide_info.py

```diff
@@ -1,20 +1,21 @@
 """Get Slide Meta Data information"""
 from tiatoolbox.dataloader import wsireader
 from tiatoolbox.decorators.multiproc import TIAMultiProcess
+from tiatoolbox.utils.exceptions import FileNotSupported
 
 import os
 
 
 @TIAMultiProcess(iter_on="input_path")
-def slide_info(input_path, output_dir=None):
-    """Single file run to output or save WSI meta data.
-
-    Multiprocessing uses this function to run slide_info in parallel
+def slide_info(input_path, output_dir=None, verbose=True):
+    """Returns WSI meta data. Multiprocessing decorator runs this function in parallel.
 
     Args:
         input_path (str): Path to whole slide image
         output_dir (str): Path to output directory to save the output
+        verbose(bool): Print output, default=True
```

Comment:
space after verbose

Reply:
Fixed in this commit https://github.com/TIA-Lab/tiatoolbox/commit/3ac21d1b573dc9f272a8527ef14e866b52c9c631